testhelper/integrationtesthelper.go

```diff
@@ -57,7 +57,7 @@ type TimeTracker struct {
 	Mu *sync.Mutex
 }
 
-// IntegrationTimedLidarSensor returns a mock timed lidar sensor
```

Comment:
Integration tests will be changed in this ticket: https://viam.atlassian.net/browse/RSDK-4863